CHANGELOG.md

```diff
@@ -11,8 +11,11 @@ All notable changes to this project will be documented in this file.
 N/A
 
 ### Enhancements
-N/A
-
+- New **NSAttributedString** extensions
```

Comment:
I mean something like this:
[New Array Extensions]
- extension 1
- extension 2
- extension 3 #X by @someone
[New String Extensions]
- extension 1 #X by @someone

The link to the PR would kind of act like a divider. This is really not important but try to keep entries on one line if possible! ❤️